modin/experimental/core/io/glob/__init__.py

```diff
@@ -11,4 +11,4 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
-"""Experimental Pickle format type IO functions implementations."""
+"""Experimental Glob format type IO functions implementations."""
```

Comment:
We do not have Glob format. This should be probably fixed.

Reply:
Not fixed yet.

Reply:
Did not save changes in the file :(